dotcom-rendering/src/web/browser/bootCmp/init.ts

```diff
@@ -4,10 +4,9 @@ import {
 	onConsentChange,
 } from '@guardian/consent-management-platform';
 import type { ConsentState } from '@guardian/consent-management-platform/dist/types';
-import { getCookie, log } from '@guardian/libs';
+import { getCookie, loadScript, log } from '@guardian/libs';
```

Comment:
Not sure why it wasn't working before, but getting no errors with this import from `@guardian/libs`